web/default/src/i18n/locales/zh.json

```
@@ -2362,10 +2370,28 @@
     "Model not found": "模型未找到",
     "Model performance metrics": "模型性能指标",
     "Model Price": "模型价格",
-    "Model Price Not Configured": "模型价格未配置",
     "Model price is not configured. Please complete model pricing in settings.": "模型价格未配置，请前往设置补充模型价格。",
+    "Model Price Not Configured": "模型价格未配置",
     "Model prices": "模型价格",
     "Model prices reset successfully": "模型价格重置成功",
+"Configure billing mode for image generation models. \"Per-token\" uses the standard token ratio. \"Per-resolution\" charges a flat price per image based on output size (1K ≤ 1024px, 2K ≤ 2048px, 4K > 2048px).": "配置图片生成模型的计费方式。「按 Token」使用标准 Token 比率计费；「按分辨率」根据输出尺寸（1K ≤ 1024px、2K ≤ 2048px、4K > 2048px）按张收取固定费用。",
+"Leave price fields empty to use the built-in default prices. Custom prices override the defaults.": "价格字段留空则使用内置默认价格，填写后将覆盖默认值。",
+"Billing mode": "计费方式",
+"Per-resolution": "按分辨率",
```

Biome (2.4.15)

[error] The key Per-resolution was already declared.

(lint/suspicious/noDuplicateObjectKeys)